### ISSUE TITLE
test_thurstone_scaling failures with SciPy 1.14

### ISSUE THREAD

**Issue by ginggs** [hoechenberger/questplus#55]
Since upgrading to SciPy 1.14 in Debian, we are seeing `test_thurstone_scaling` failing on some architectures (ARM, POWER and RISC-V, among others).

```
>           assert np.isclose(next_stim_1, expected_stim_1)
E           assert False
E            +  where False = <function isclose at 0xffff9d268ef0>(1.0, 0.6)
E            +    where <function isclose at 0xffff9d268ef0> = np.isclose
```
 
The new failure occurs when `trial_idx` is 21, so I hoped the following change would resolve it:

```
-        if trial_idx in (6, 20):
+        if trial_idx in (6, 20, 21):
```

However, at the time of the failure, the values are:

```
next_stim_1:  1.0
expected_stim_1:  0.6
next_stim_2:  0.6000000000000001
expected_stim_2:  1.0
```

And  the following change was also needed:

```
-                expected_stim_1 == next_stim_2 and
-                expected_stim_2 == next_stim_1
+                np.isclose(expected_stim_1, next_stim_2) and
+                np.isclose(expected_stim_2, next_stim_1)
```

**Comment by hoechenberger** [comment]
Thanks @ginggs! Would you like to submit a PR with the fix?

**Comment by ginggs** [comment]
Sure!  I have submitted #56

**Comment by hoechenberger** [comment]
Closed via #56
